flink-runtime/src/test/scala/org/apache/flink/runtime/testingUtils/TestingMemoryArchivist.scala

```diff
@@ -31,10 +31,11 @@ trait TestingMemoryArchivist extends ActorLogMessages {
 
   def receiveTestingMessages: Receive = {
     case RequestExecutionGraph(jobID) =>
-      graphs.get(jobID) match {
-        case Some(executionGraph) => sender ! ExecutionGraphFound(jobID, executionGraph)
-        case None => sender ! ExecutionGraphNotFound(jobID)
+      val executionGraph = getGraph(jobID)
+      if (executionGraph != null) {
```

Comment:
I like @tillrohrmann to use Option as alternative to null. In Java land, Guava's Optional could be use to do similar thing (which I think will be part of Java8)